cita-chain/core/Cargo.toml

```diff
@@ -1,22 +1,21 @@
 [package]
 name = "core"
 version = "0.1.0"
-authors = ["Cryptape Technologies <contact@cryptape.com>", "Parity Technologies <admin@parity.io>"]
+authors = ["Cryptape Technologies <contact@cryptape.com>"]
```

Comment:
There'are still codes about `filter` of `Parity`.

But, it's fine.

Reply:
Just let it go.